app/views/users/_account_providers_emails.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="crayons-card mb-6 p-5" id="emails">
+  <div class="crayons-card__header">
+    <h2 class="crayons-card__headline">Account emails</h2>
+  </div>
```

Comment:
I think you can drop the card inner styling for sake of consistency with rest of the sections in settings:

```suggestion
  <h2 class="mb-2">Account emails</h2>
```